app/models/store_working_hour.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class StoreWorkingHour < ApplicationRecord
+  self.table_name = 'stores_working_hours'
+
+  validates :day, presence: true
```

Comment:
we should have a map of valid day just to prevent some weird string in may occured in day